src/error.rs

```diff
@@ -153,10 +150,11 @@ impl From<InternalFrom<Error>> for Error {
 }
 
 impl<T> From<InternalFrom<T>> for Error
-where T: Into<Kind> {
+    where T: Into<Kind>
+{
```

Comment:
This is the default style for rustfmt. There is a newer style (follow the rust style RFCs) that has where not indented, but each of the where clauses on new lines and indented. I've turned that style on and updated.